run-nitro-enclaves-acm-tests

```diff
@@ -38,21 +101,164 @@ function test_acm_ne_files() {
+
+        out=$($P11NE_CLI init-token \
+                --key-db $KEY_DB \
+                --label $TOKEN_LABEL \
+                --pin $TOKEN_PIN)
+        echo $out | grep "\"Ok\": \"None\"" > /dev/null || ERRORS="$ERRORS $ERROR_INIT_TOKEN"
+
+        out=$($P11NE_CLI describe-device)
+        ttl_init=$($P11NE_CLI describe-device \
+                | jq '.Ok.DeviceDescription.tokens[] | select(.label == "test-token")' \
+                | grep "ttl_secs" \
+                | sed -e "s/[[:space:]]\+/ /g" \
+                | cut -d' ' -f3)
+
+        out=$($P11NE_CLI describe-token \
+                --label $TOKEN_LABEL \
+                --pin $TOKEN_PIN)
+
+        key_uri=$(echo $out | jq '.' | grep "\"uri\"" | cut -d'"' -f4)
+        key_uri_base=$(echo $key_uri | cut -d' ' -f 2 | cut -d'"' -f2 | rev | cut -d'=' -f2- | rev)
+
+        # Test sign / verify
+        echo "Sign This" > input
+        openssl dgst \
+                -keyform engine \
+                -engine pkcs11 \
+                -sign "${key_uri_base}=private?pin-value=${TOKEN_PIN}" \
+                -out test.sig input 2> /dev/null
+        out=$(openssl dgst \
+                -keyform engine \
+                -engine pkcs11 \
+                -verify "${key_uri_base}=public" \
+                -signature test.sig input 2> /dev/null)
+        echo $out | grep "Verified OK" > /dev/null || ERRORS="$ERRORS $ERROR_VERIFY"
+
+	# Test encrypt / decrypt
+        echo "Encrypt This" > input
+        openssl pkeyutl \
+                -keyform engine \
+                -engine pkcs11 \
+                -encrypt \
+                -pubin \
+                -inkey "${key_uri_base}=public" \
+                -out test.crypt -in input 2> /dev/null
+        openssl pkeyutl \
+                -keyform engine \
+                -engine pkcs11 \
+                -decrypt \
+                -inkey "${key_uri_base}=private?pin-value=${TOKEN_PIN}" \
+                -in test.crypt > test.decrypt 2> /dev/null
+        out=$(diff input test.decrypt)
+        [[ -z $out ]] || ERRORS="$ERRORS $ERROR_DECRYPT"
+
+        # Test refresh token
+        out=$($P11NE_CLI refresh-token --label $TOKEN_LABEL --pin $TOKEN_PIN)
+        echo $out | grep "\"Ok\": \"None\"" > /dev/null || ERRORS="$ERRORS $ERROR_REFRESH"
+
+        out=$($P11NE_CLI describe-token --label $TOKEN_LABEL --pin $TOKEN_PIN)
+        ttl_end=$(echo $out | jq '.' | grep "ttl_secs" | sed -e "s/[[:space:]]\+/ /g" | cut -d' ' -f3 | sed "s/.$//")
+
+        [[ $(( $ttl_end + $EPS )) -ge $ttl_init ]] || ERRORS="$ERRORS $ERROR_REFRESH"
+
+        # Release token
+        out=$($P11NE_CLI release-token --label $TOKEN_LABEL --pin $TOKEN_PIN)
+        echo $out | grep "\"Ok\": \"None\"" > /dev/null || ERRORS="$ERRORS $ERROR_RELEASE"
+        out=$($P11NE_CLI describe-token --label $TOKEN_LABEL --pin $TOKEN_PIN)
+        echo $out | grep "\"Err\": \"TokenNotFound\"" > /dev/null || ERRORS="$ERRORS $ERROR_RELEASE"
+
```

Comment:
Let's also add here a block that: generates secp384r1 EC key, initializes a token, does a sign/verify like above, refresh and release.